packages/manager/src/components/application/ApplicationIndex.tsx

```diff
@@ -13,10 +16,26 @@ const styles = StyleSheet.create({
   },
 });
 
-export default () => {
+export default withRouter(({ history }) => {
   const location = useLocation();
   const { applicationName } = useParams();
+  const [environment] = useState<IEnvironment>(config.environments[0]);
 
+  const onClickConfirm = async () => {
+    await deleteApplication(environment, applicationName);
+    history.push("/applications");
+  };
+
+  const titleToolbar = (
+    <ConfirmButton
+      label="Delete"
+      title={`Delete Application "${applicationName}"`}
+      variant={ButtonVariant.danger}
+      onConfirm={() => onClickConfirm()}
```

Comment:
If the `onClickConfirm` already use arrow function, not more arrow function should be used inline
```suggestion
      onConfirm={onClickConfirm}
```

Reply:
Ahh. Nice Catch! I missed this while refactoring my code.